common/jwt.js

```diff
@@ -45,15 +45,60 @@ export const jwtFromExchange = async (exchange, rp) => {
   const header = {
     alg: signingKey.type,
     typ: 'JWT',
-    kid: signingKey.id
+    kid: signingKey.id,
   };
 
+  // Handle DC API workflow differently
+  if(rp.workflow?.type === 'dc-api' && exchange.variables?.dcApiResponse) {
+    try {
+      const now = Math.floor(Date.now() / 1000);
+      const dcApiResponse = exchange.variables.dcApiResponse;
+      // Default to 1 hour
+      const expirySeconds = rp.idTokenExpirySeconds || 3600;
+
+      const subject =
+        dcApiResponse.response['org.iso.18013.5.1'].document_number;
+
+      const verified =
+        dcApiResponse.response.issuer_authentication == 'Valid' &&
+        dcApiResponse.response.device_authentication == 'Valid';
+
+      const errors = dcApiResponse.response.errors;
+
+      const payload = {
+        iss: config.server.baseUri,
+        aud: rp.clientId,
+        sub: subject || exchange.id,
```

Comment:
Need to identify what the subject should be. It may not be guaranteed that the document number would be requested by a relying party.

Need to identify if this JWT construction is correct.